README.md

```diff
@@ -387,6 +387,7 @@ Table of Contents
   * [Directus](https://directus.io) — Headless CMS. A completely free and open-source platform for managing assets and database content on-prem or in the Cloud. No limitations or paywalls.
   * [Forestry.io/](https://forestry.io/) — Headless CMS. Give your editors the power of Git. Create and edit Markdown-based content with ease. Comes with three free sites that includes 3 editors, Instant Previews. Integrates with blogs hosted on Netlify/GitHubpages/ elsewhere
   * [FrontAid](https://frontaid.io/) — Headless CMS that stores JSON content directly in your own Git repository. No restrictions.
+  * [Joomla!](https://launch.joomla.org/) - Joomla! is an award-winning content management system (CMS), which enables you to build web sites and powerful online applications. Developed and supported since 2005 it recently released Joomla 4,and is free to download and use. 
```

Comment:
Best then to update the description with features and limitations of the free hosting.